lib/subscriptions_test_kit/suites/subscriptions_r5_backport_r4_client/workflow/interaction_verification/event_notification_verification_test.rb

```diff
@@ -34,6 +34,15 @@ class EventNotificationVerificationTest < Inferno::Test
                                      Inferno only supports: #{ALLOWED_MIME_TYPES.map { |type| "'#{type}'" }.join(', ')}.
                                      Event notification was sent with Content-Type: '#{DEFAULT_MIME_TYPE}'.))
           end
+
+          # Verification for hl7.fhir.uv.subscriptions_1.1.0@26
+          assert(request.content_type_header == mime_type,
+                 'Content type of request does not match the Subscription MIME type')
+          # Verification for hl7.fhir.uv.subscriptions_1.1.0@27
+          subscription&.channel&.header&.each do |sc_header|
+            assert(request.header.include?(sc_header),
+                   'Subscription channel header is not conveyed as HTTP request header')
+          end
```

Comment:
take out these changes. These are notifications that Inferno sent, so we don't need to check them.